src/components/command-bar/commands/search/components/integrations-tab-contents/index.tsx

```diff
@@ -6,7 +6,7 @@
 import { useMemo } from 'react'
 import { IconPipeline16 } from '@hashicorp/flight-icons/svg-react/pipeline-16'
 import { useIntegrationsByProductSlugs } from 'hooks/integrations/use-integrations-by-product-slugs'
-import { getFilteredIntegrations } from 'views/product-integrations-landing/components/searchable-integrations-list/helpers/get-filtered-integrations'
+import { getFilteredIntegrations } from 'views/product-integrations-landing/components/searchable-integrations-list/helpers'
```

Comment:
🗒️ This change goes alongside adding an `index.ts` barrel file to this `helpers` folder.